godtools/Configurations/TractBindings.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TractBindings.swift
+//  godtools
+//
+//  Created by Devserker on 5/24/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+class TractBindings: NSObject {
+    
+    @nonobjc static var bindings = [String: BaseTractElement]()
+    @nonobjc static var dismissBindings = [String: BaseTractElement]()
+    
+    static func setupBindings() {
+        TractBindings.bindings = [String: BaseTractElement]()
+        TractBindings.dismissBindings = [String: BaseTractElement]()
+    }
+    
+    static func addBindings(_ element: BaseTractElement) {
+        TractBindings.addOpeningBindings(element)
+        TractBindings.addDismissBindings(element)
+    }
+    
+    static func addOpeningBindings(_ element: BaseTractElement) {
+        if element.elementListeners() == nil {
```

Comment:
Not a big deal as the code is still readable, but this and the next function could both be `guard let` statements. You don't have to change this, but you can if you want to :)